src/components/Dashboard/DashboardPage.js

```diff
@@ -124,30 +129,145 @@ function DashboardPage() {
             <div style={{ marginTop: '0rem' }}>
               <ul>
                 <li>
-                  <Button
-                    onClick={() => {
-                      console.log('Clicked')
+                  <div
+                    style={{
+                      display: 'grid',
+                      gridTemplateColumns: '1fr 3fr',
+
+                      gridColumnGap: '1rem',
+                      padding: '1rem 0rem',
                     }}
-                    wide
                   >
-                    DASHBOARD
-                  </Button>
+                    <div
```

Comment:
This left-side Panel divs are different in Arbitrations and Dashboard page. This should be uniform throughout the project and reusable. The panel should not be called from every page but only maintain in any root file similar to the earlier `MenuPanel.js` which was called only in `MenuView.js`.

Reply:
1. I am aware of the issue, I am creating one parent component for the side panel as well as the header architecture, which would render all the children (dashboard/profile/arbitrations).

2. The difference in the panel is due to the fact we are moving away from the button based panel(arbitrations) to div(dashboard) based panel, I will address this in the next push.